spec/GraphQLOverHTTP.md

```diff
@@ -426,8 +426,8 @@ In alignment with the
 specification, when a client does not include at least one supported media type
 in the `Accept` HTTP header, the server MUST either:
 
-1. Disregard the `Accept` header and respond with the default media type of
-   `application/json`, specifying this in the `Content-Type` header; OR
+1. Disregard the `Accept` header and respond with the server's choice of media
+   type, indicating this in the `Content-Type` header; OR
```

Comment:
Would a suggestion make sense here? (if you're not sure what to choose, use `application/json`)

Reply:
The issue is that if you have a query that has deferred parts in it you have to respond with `MultiPart/Mixed` or an event stream.

Reply:
I think there’s enough context in the rest of the spec to infer the suggestion.